[example] adapt can example from stm32f3 to f469

se-bi opened the pull request:
retransmit incoming messages and increase idx
LED color toggling (red activity/green received message)
fix CanFilter bank for Can2
enable can1 to use can2 ...

salkinium commented on examples/stm32f469_discovery/can/main.cpp:
```diff
@@ -0,0 +1,126 @@
+	RCC->APB1ENR  |=  RCC_APB1ENR_CAN1EN;
+	/* do not try this, crashes uC:
+	Can1::initialize<Board::systemClock, Can1::Bitrate::kBps125>(9);
+	*/
```
This issue is tracked in #71.

chris-durand committed:
[stm32] CAN: Regard instance hardware interdependencies in driver

se-bi committed:
[example] adapt can example from stm32f3 to f469

retransmit incoming messages and increase idx
LED color toggling (red activity/green received message)
fix CanFilter bank for Can2

se-bi commented:
I rebased this branch on the fix/stm32_can by @chris-durand,
removed the manual initialization of CAN1,
tested successful on DISCO F469 and F4 (adapted similar example).
